candi/bashible/common-steps/node-group/051_pull_kubernetes_api_proxy.sh.tpl

```diff
@@ -13,7 +13,7 @@
 # limitations under the License.
 
 if docker version >/dev/null 2>/dev/null; then
-  docker pull {{ printf "%s%s:%s" $.registry.address $.registry.path (index $.images.controlPlaneManager "kubernetesApiProxy") }}
+  HOME=/ docker pull {{ printf "%s%s:%s" $.registry.address $.registry.path (index $.images.controlPlaneManager "kubernetesApiProxy") }}
```

Comment:
Why do we need to set HOME env? We [have docker pull](https://github.com/deckhouse/deckhouse/blob/main/candi/bashible/bundles/ubuntu-lts/node-group/035_pull_pause.sh.tpl#L23-L24) in another step without any HOMe env.